build/index.js

```diff
@@ -0,0 +1,168 @@
+"use strict";
```

Comment:
the content of the `build` folder should be ignored

Reply:
Sure, I'll exclude the build folder